src/pages/groupComparison/OverlapExclusionIndicator.tsx

```diff
@@ -12,174 +13,168 @@ export interface IOverlapExclusionIndicatorProps {
-                    alertClass = "alert-info";
+            // handle survival mode
+            message = makeSurvivalTabMessage(count);
+        } else {
+            // determine groups
+            const includedGroups = selectionInfo.groups.filter(g=>!(g.uid in selectionInfo.excludedFromAnalysis));
+            const groupsAreExcluded = includedGroups.length < selectionInfo.groups.length;
+            let groupsSummary;
+            if (groupsAreExcluded) {
+                groupsSummary = <span>between {joinGroupNames(includedGroups, "and")}</span>;
+            } else {
+                groupsSummary = <span>in the selected groups</span>;
+            }
 
-                    if (existOverlapping) {
-                        const singular = (
-                            (this.props.only === "sample" && selectionInfo.overlappingSamples.length === 1) ||
-                            (this.props.only === "patient" && selectionInfo.overlappingPatients.length === 1)
-                        );
-                        message = (
-                            <span>
-                                {caseCountsSummary}
-                                {` ${singular ? "is" : "are"} excluded from `}
-                                {!this.props.overlapTabMode ? `this analysis.` : `analysis in other tabs.`}
-                            </span>
-                        );
+            message = (
+                <span>
+                    {`${caseType === "sample" ? "Samples" : "Patients"} (${count}) that overlap `}
+                    {groupsSummary}
+                    {` are ${this.props.store.overlapStrategy === OverlapStrategy.INCLUDE ? "included in" : "excluded from"}`}
+                    {this.props.overlapTabMode ?
+                        ` ${caseType}-level analysis in other tabs.`:
+                        ` this ${caseType}-level analysis.`
                     }
+                </span>
+            );
+        }
 
-                    if (excludedGroups.length > 0) {
-                        excludedGroupsSummary = (
-                            <span>
-                                {joinGroupNames(excludedGroups, "and")}&nbsp;{excludedGroups.length === 1 ? "is" : "are"}
-                                {` completely overlapping with other selected groups, so `}
-                                {this.props.overlapTabMode ?
-                                    `${excludedGroups.length === 1 ? "is" : "are"} excluded from analysis in other tabs.` :
-                                    `${excludedGroups.length === 1 ? "has" : "have"} been excluded from this analysis.`
-                                }
-                            </span>
-                        );
-                    }
-                    break;
+        return (
+            <div>
+                <i
+                    className={`fa fa-md ${this.classNames.icon}`}
+                    style={{
+                        color: "#000000",
```

Comment:
isn't this default color?

Reply:
true

Reply:
I was just about to take it out but then I thought, theres some chance that errant CSS could target this, so maybe it makes sense to make it explicit here?

Reply:
I mean, if it's part of the class of things "text" then it should be default.  If not then ....